### ISSUE TITLE
A11y - Contact Page

### ISSUE THREAD

**Issue by luistoptal** [gigascience/gigadb-website#1376]
## User story

- Missing heading levels: Page has only h4, no h1, etc
- Breadcrumbs: Low contrast
- Map image: Img missing alt text
- Text Content: Low contrast links
- Address: Use semantic html, e.g. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address

https://beta.gigadb.org/site/contact

## Acceptance criteria

- [x] There is a h1 heading and any other heading do not skip heading levels
- [ ] ~~Breadcrumbs have enough contrast~~ Fixed in #1427 
- [x] Map image has alt attribute
- [ ] ~~Text link has enough contrast~~ Fixed in #1427 
- [x] Address uses semantinc html

**Comment by luistoptal** [comment]
Relevant file protected/views/site/contact.php

**Comment by luistoptal** [comment]
Handle breadcrums in this other ticket as they are a common feature in many pages: https://github.com/gigascience/gigadb-website/issues/1390

**Comment by luistoptal** [comment]
Fix text color as part of a separate ticket that handles color contrast https://github.com/gigascience/gigadb-website/issues/1363
